### PR TITLE
Icebox Wendigo Buff + Crusher Trophies

### PR DESCRIPTION
**Pull request by Whoneedspacee**
## About The Pull Request

Adds a new megafauna the clockwork defender, it's essentially between normal mobs and lavaland elites, not really having special abilities of its own besides above average stats.

Adds Godslayer armor, if you defeat all 3 icebox megafauna you get a cool looking armor set which heals you the first time you get crit every 10 minutes, I get this is a new addition maybe it can just be a resprited H.E.C.K. suit if you want, but flavor wise it looks cool for icebox and someone made the sprite for me.

Major rework to the wendigo in making him much more difficult and unique, he doesn't have any loot changes but he is much more difficult and I'd consider him the wall to the godslayer armor.

Adds crusher trophies to most icebox mobs.

## Why It's Good For The Game

The godslayer armor gives a nice incentive to defeat the wendigo since as he is he doesn't really have any especially good drops of his own, it'd be cool if this could exist or there could be some other loot additions to him. I think it's a cool cap off since you have to defeat all 3 megafauna to craft it as well, which is basically impossible in a shorter round.

The wendigo changes make him a much better fight, before this I wouldn't really even have considered him a megafauna since 3 range mods essentially meant that you had already won.

Crusher trophies for all mobs should have been there from the start, crusher on icebox has not been kind since it has been missing all of the lavaland trophies, this fixes that problem.

## Changelog
:cl: Whoneedspacee
add: Icebox mobs now have crusher trophies.
add: Godslayer Armor can now be crafted by gathering drops from all three icebox megafauna.
add: Adds the easy clockwork defender megafauna to the asteroid ruin.
balance: The Wendigo is now a proper megafauna.
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
you should probably atomize your PRs a bit more

File: code/game/turfs/closed/minerals.dm
Comment (Qustinnus) [inline code/game/turfs/closed/minerals.dm:95]: This is too fast and should use the time defines. That goes for most of the code, seems to be a lack of time defines (using SECONDS define)

Comment (Whoneedspacee) [reply code/game/turfs/closed/minerals.dm:95]: Ah sorry most of this is old and using the old style of megafauna sleep type stuff if that makes sense.

**Comment by LemonInTheDark** [comment]
Jesus that's a lot, has potential tho

**Comment by Jared-Fogle** [comment]
this 100% needs to be atomized

**Comment by D4C-420** [comment]
> The tendril equivalent in icebox the demonic portal now has a gps signal.

> You can now mine mineral turfs that are marked as "weak" with your hands, this is used for icebox snow to prevent people getting stuck in holes permanently.

> Drake armor now has cold protection equal to the normal mining explorer suit so people who make drake armor on icebox don't freeze while outside.

Yes. Real quality shit right there.

**Comment by ShibaInuLord** [comment]
in icondiff it looks like you deleted the clockwork chaplain armour

**Review by Mothblocks — CHANGES_REQUESTED** [review]
atomize this

**Comment by Timberpoes** [comment]
As Jared said, atomise this.

We can merge the simple stuff and discuss the less-than-simple stuff which aren't "QoL" changes are are infact feature additions and/or balance changes.

**Comment by LemonInTheDark** [comment]
@Timberpoes @Jared-Fogle this more to your taste?

**Comment by Jared-Fogle** [comment]
In terms of atomization, yes.

**Comment by Whoneedspacee** [comment]
yeah wait for the other one to be merged ofc

**Review by Twaticus — APPROVED** [review]

File: code/datums/components/crafting/recipes.dm
Comment (Mothblocks) [inline code/datums/components/crafting/recipes.dm:827]:
```suggestion
		/obj/item/wendigo_skull = 1,
		/obj/item/clockwork_alloy = 1)
```

Comment (Mothblocks) [reply code/datums/components/crafting/recipes.dm:827]: No alignment indenting.

File: code/datums/status_effects/buffs.dm
Comment (Mothblocks) [inline code/datums/status_effects/buffs.dm:525]: Named argument.

File: code/datums/status_effects/buffs.dm
Comment (Mothblocks) [inline code/datums/status_effects/buffs.dm:529]: Named argument.

File: code/modules/clothing/suits/cloaks.dm
Comment (Mothblocks) [inline code/modules/clothing/suits/cloaks.dm:136]:
```suggestion
	if(loc == user && world.time <= effect_cooldown)
```

File: code/modules/clothing/suits/cloaks.dm
Comment (Mothblocks) [inline code/modules/clothing/suits/cloaks.dm:117]: Use the cooldown defines.

File: code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm
Comment (Mothblocks) [inline code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:172]: Better variable name. In this case, shockwave.

Comment (Mothblocks) [reply code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:172]: Also remove the `src.`.

File: code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm
Comment (Mothblocks) [inline code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:222]: Here too.

File: code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm
Comment (Mothblocks) [inline code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:217]: Make this a define.

Comment (Mothblocks) [reply code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:217]: Make every number you've done this to a define as well.

File: code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm
Comment (Mothblocks) [reply code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm:222]: And everywhere else, in terms of better variable names and removing `src.`.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Sorry that nobody's been reviewing this, I'll take up the mantle.

**Comment by Whoneedspacee** [comment]
> the changelog doesnt include ANYTHING about an entire new megafauna and armor... neither does the titlte

The changelog is for stuff thats relevant and this is why it wasnt referenced

>Adds a new megafauna the clockwork defender, it's essentially between normal mobs and lavaland elites, not really having special abilities of its own besides above average stats.

Its just a random fuckoff mob used for the armor set essentially, theres no way I'd put a real megafauna on the top level of the station.

As for the armor set, I could put it in the changelog but eh? I don't think its particularly interesting, but I'll include it because you asked.

**Comment by Mothblocks** [comment]
> The changelog is for stuff thats relevant and this is why it wasnt referenced

...Huh? A new boss is certainly something that should be in the changelog. The changelog is for player facing changes, so if a player can see it, it should be in the changelog.

**Comment by Mothblocks** [comment]
Update the changelog (unless the content you added is impossible to see for players without admin intervention) and this is good to go.

**Comment by IndieanaJones** [comment]
I like the clockwork sprite, sad you didn't give it anything unique.

**Comment by Whoneedspacee** [comment]
> I like the clockwork sprite, sad you didn't give it anything unique.

I would’ve but just trying to avoid adding much new content since the maintainers don’t care for it.

**Comment by Whoneedspacee** [comment]
Done.

**Pull request merged**
